### PR TITLE
Adds ansible bundle

### PR DESCRIPTION
**Pull request by tahmidefaz**

### CODE REVIEW

**Comment by tahmidefaz** [comment]
Looks like they don't have a deploy repo setup already. Hence, there's no mention of it. Let me know if I need to change anything!

File: main.yml [hunk 1]
```diff
@@ -1,3 +1,12 @@
+ansible:
+  title: Ansible
+  frontend:
+    title: Ansible
```
Comment (kruai) [inline main.yml]: `frontend.title` is an override; you only need it if you need the UI to display the name differently than the main `title` field.

**Review by kruai — CHANGES_REQUESTED** [review]
You might only need the title change, but I'm not going to approve just yet; I'm debugging an issue with Ryan F and the config in your PR looks very similar. Once I figure out what's happening with `/migrations`, I'll approve or request more changes

**Review by ryandillinfelton — COMMENTED** [review]
Looks fine to me. I'll come take a look again after we figure out what's going on with the migrations bundle so we can make sure it won't happen with this too

**Review by kruai — APPROVED** [review]
Now that we know migration-analytics is actually correct, I think this one is going to work fine.

**Review by ryelo — APPROVED** [review]

**Review by ryandillinfelton — APPROVED** [review]
LGTM

**Comment by kruai** [comment]
Your wish is my command

**Pull request merged**